### ISSUE TITLE
Subtitles block the play button on nico nico douga videos

### ISSUE THREAD

**Issue by killergerbah** [math4origami/nicokara#1]

**Comment by math4origami** [comment]
Yeah, atm you can get around that by hiding the subtitles by pressing the "s" button in the top left or "s" key on your keyboard.  Is it possible in CSS/html to let clicks go through?

**Comment by killergerbah** [comment]
just found this:
http://stackoverflow.com/questions/3680429/click-through-a-div-to-underlying-elements

**Comment by math4origami** [comment]
Wow, cool, thanks, Ill add that.  Itll mean you cant select the subs, but theres another page for that anyway XD.  Btw, a newer version is currently hosted at www.karaqueue.com.  I havent pushed to github in a while, but i guess ill update it since someone is looking ^^;
